src/components/FilterButton/FilterButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export const FilterButton = (props) => {
+  const { onClick, selectedFilter, children } = props;
+
+  return (
+    <li>
+      <a
+        href={`#/${selectedFilter === 'all' ? '' : selectedFilter}`}
+        className={cx({ selected: selectedFilter === children.toLowerCase() })}
```

Comment:
!!!do not use content(children) in your logic

Reply:
OK